software/interface.md

```diff
@@ -124,11 +124,13 @@ You can also flash a full DAPLink image to the <span class="v2">V2</span> device
 
 [Download latest full DAPLink image](/docs/software/assets/DAPLink-factory-release/kl27z_bl_0255_if_0255_microbit_full_image.bin){: .btn.sm-btn download}
 
-You will need to register for and download the [**Bootloader Host Application (blhost)**](https://www.nxp.com/design/software/development-software/mcuxpresso-software-and-tools-/mcuboot-mcu-bootloader-for-nxp-microcontrollers:MCUBOOT?&tab=Design_Tools_Tab) from NXP. IN the /bin folder you will find executables for your operating system.
+You will need to register for and download the [**Bootloader Host Application (blhost)**](https://www.nxp.com/design/software/development-software/mcuxpresso-software-and-tools-/mcuboot-mcu-bootloader-for-nxp-microcontrollers:MCUBOOT?&tab=Design_Tools_Tab) from NXP. In the `/bin` folder you will find executables for your operating system.
+
+<div class="alert alert-warning">Please ensure you are using the latest version of blhost, old versions incompatibility with new OS versions can have unexpected results.</div>
```

Comment:
Actually, I need to update this warning to something else before we can merge this PR, but everything else is good to review.

Reply:
Okay, I've updated this warning to have the right content.